programs/ssl/ssl_client2.c

```diff
@@ -726,6 +737,8 @@ int main( int argc, char *argv[] )
             opt.crt_file = q;
         else if( strcmp( p, "key_file" ) == 0 )
             opt.key_file = q;
+        else if( strcmp( p, "key_opaque" ) == 0 )
```

Comment:
I don't think we should parse an option if it's not advertised in the `USAGE` string. Could you guard this by `MBEDTLS_USE_PSA_CRYPTO`, too?

Reply:
We do parse all the other options even if they're not advertised in the usage string, and initially I was inclined to just to the same for homogeneity's sake. However, I agree that refusing to parse options we don't support would be a good think, so we can start adding that to new options and fix the old ones later on.